bootstrap.sh

```diff
@@ -55,9 +55,9 @@ dokku_install_package() {
   apt-get update -qq > /dev/null
 
   if [[ -n $DOKKU_CHECKOUT ]]; then
-    apt-get install dokku=$DOKKU_CHECKOUT
+    apt-get install -y dokku-$DOKKU_CHECKOUT
   else
-    apt-get install dokku
+    apt-get install -y dokku
```

Comment:
'apt-get install dokku' was prompting a question. So, I changed it. This is not relevant to the issue. But I faced with this while debugging.